src/main/java/by/itech/lab/supplier/repository/UserRepository.java

```diff
@@ -19,6 +19,11 @@
 
     Optional<User> findById(Long id);
 
+    Optional<User> findByEmail(String email);
+
+    @Query("select c from User c where :active is null or c.active=:active")
```

Comment:
What is the goal of :active is null condition?

Reply:
The method in UserController may receive active=null|true|false conditions, meaning to get all users, inly active users and not active users